packages/ui/src/council/queries/council.graphql

```diff
@@ -0,0 +1,24 @@
+fragment CouncilMemberFields on CouncilMember {
+  id
+  member {
+    ...MemberFields
+    councilMembers {
+      __typename
+    }
```

Comment:
This looks odd. Why do we need to query this? I couldn't find where it was used.

Reply:
So far the only way I found to get all the `numberOfTerms` in one request (number of time a member was elected council), is to count the `member.councilMembers`. So there's no need for any real data in the `councilMembers`

Reply:
Ah right, thanks!